packages/forms/demo/stories/common.tsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Label, Hint, Message, IMessageProps } from '@zendeskgarden/react-forms';
+
+export const labelArgTypes = {
+  label: { name: 'children', table: { category: 'Label' } },
+  isLabelRegular: { name: 'isRegular', table: { category: 'Label' } },
+  isLabelHidden: { name: 'hidden', table: { category: 'Label' } }
+};
+
+export const hintArgTypes = {
+  hasHint: { name: 'Hint', table: { category: 'Story' } },
+  hint: { name: 'children', table: { category: 'Hint' } }
+};
+
+export const messageArgTypes = {
+  hasMessage: { name: 'Message', table: { category: 'Story' } },
+  message: { name: 'children', table: { category: 'Message' } },
+  validationLabel: {
+    control: { type: 'text' },
+    table: { category: 'Message' }
+  },
+  validation: {
+    options: ['success', 'warning', 'error'],
+    control: { type: 'radio' },
+    table: { category: 'Message' }
+  }
```

Comment:
The `validation` arg should not always be moved here. Only in the cases where the top-level form component does _not_ expose a `validation` prop. For example, this "steals" the `validation` prop from the `Input` component.

Reply:
Gotcha, I updated this to allow the original `validation` arg to be part of the main story args.